### DISCUSSION TITLE
Alternative installations

### DISCUSSION THREAD

**Discussion by willichan** [SigNoz/signoz#3769] [Q&A]
Signoz appears to have the features I am looking for, both for my home network, and for the office.  I am, however, neither using Docker nor Kubernetes in either environment.

Are there any alternative setup options/instructions for installing Signoz, such as VMDK, flat-pack, or manual install?

I am using Proxmox for my home environment, and Nutanix in the office.

**Comment by pranay01** [comment]
@willichan Can you share what do you mean by manual install in this context?

A related issue : https://github.com/SigNoz/signoz/issues/392

**Reply by willichan** [reply]
By manual install, I am referring to installing and configuring supporting packages, and either building from code or copying binaries to their respective folders and creating necessary configuration files, to have a functioning system.  In other words, reproducing the work done in order to create a working container image, but without the container or image.

**Reply by willichan** [reply]
The issue in #392 does relate, and would take care of my issue as well.
